React/Views/RCTConvert+MapKit.m

```diff
@@ -66,4 +66,42 @@ + (RCTPointAnnotation *)RCTPointAnnotation:(id)json
 
 RCT_ARRAY_CONVERTER(RCTPointAnnotation)
 
++ (RCTPolyline *)RCTPolyline:(id)json
+{
+
+  json = [self NSDictionary:json];
+  NSArray* locations = (NSArray*)json[@"locations"];
+
+
+  CLLocationCoordinate2D polylineCoordinates[locations.count];
+  NSUInteger index = 0;
+  for (NSDictionary* location in locations) {
+    CLLocationCoordinate2D coordinate;
+
+    coordinate.latitude = [location[@"lat"] doubleValue];
+    coordinate.longitude = [location[@"lon"] doubleValue];
+
+    polylineCoordinates[index++] = coordinate;
+  }
+
+  RCTPolyline *polyline = [RCTPolyline polylineWithCoordinates:polylineCoordinates count:locations.count];
+
+  unsigned rgbValue = 0;
+  NSScanner *scanner = [NSScanner scannerWithString:[RCTConvert NSString:json[@"strokeColor"]]];
```

Comment:
Could you point me to a usage like this in the code base?  The pattern seems to revolve around the use of 'processColor' but I can't seem to figure out how this works.